back/engines/commercial/verification/app/controllers/verification/web_api/v1/verification_methods_controller.rb

```diff
@@ -21,21 +21,21 @@ def index
 
         def first_enabled
           method = VerificationService.new.first_method_enabled
-          authorize method, policy_class: VerificationMethodPolicy if method
+          authorize method, policy_class: VerificationMethodPolicy
           respond_with(method)
         end
 
         def first_enabled_for_verified_actions
           method = VerificationService.new.first_method_enabled_for_verified_actions
-          authorize method, policy_class: VerificationMethodPolicy if method
+          authorize method, policy_class: VerificationMethodPolicy
           respond_with(method)
         end
 
         private
 
         def respond_with(method)
           if method.nil?
-            head :not_found
```

Comment:
Also changed this status code so that it doesn't gets read as an error on the FE (has some benefits like the FE not doing multiple retries)